Neos.EventSourcedContentRepository/Classes/Domain/Context/Workspace/WorkspaceCommandHandler.php

```diff
@@ -192,15 +198,6 @@ public function handleCreateRootWorkspace(CreateRootWorkspace $command): Command
         $eventStore->commit($streamName, $events);
         $commandResult = $commandResult->merge(CommandResult::fromPublishedEvents($events));
 
-        $commandResult = $commandResult->merge($this->nodeCommandHandler->handleCreateRootNode(
-            new CreateRootNode(
-                $contentStreamIdentifier,
-                $command->getRootNodeIdentifier(),
-                $command->getRootNodeTypeName(),
-                $command->getInitiatingUserIdentifier()
-            )
-        ));
-
```

Comment:
why are these lines deleted? Solved somewhere else? :)

Reply:
OK, might be a possible TODO which we need to do inside Neos at a few places. (reminder to self)